bitnami/redmine/README.md

```diff
@@ -328,6 +328,91 @@ Bitnami will release a new chart updating its containers if a new version of the
 
 Redmine writes uploaded files to a persistent volume. By default that volume cannot be shared between pods (RWO). In such a configuration the `replicas` option must be set to `1`. If the persistent volume supports more than one writer (RWX), ie NFS, `replicas` can be greater than `1`.
 
+### Deploying to a sub-URI
+
+(adapted from https://hub.docker.com/r/bitnami/redmine/)
```

Comment:
please use the github link instead, [https://github.com/bitnami/bitnami-docker-redmine](https://github.com/bitnami/bitnami-docker-redmine)

Reply:
done